source/common/common/logger.h

```diff
@@ -306,8 +306,8 @@ class Context {
   bool enable_fine_grain_logging_;
   Context* const save_context_;
 
-  std::string fancy_log_format_ = "[%Y-%m-%d %T.%e][%t][%l][%n] %v";
-  spdlog::level::level_enum fancy_default_level_ = spdlog::level::info;
+  std::string fine_grain_log_format_ = "[%Y-%m-%d %T.%e][%t][%l][%n] %v";
```

Comment:
this knowledge is duplicated somewhat duplicated here: https://github.com/envoyproxy/envoy/pull/21677/files#diff-6efc51b961fd585920d524797bb982ee6fe7d08790f7edd40af36e1de3976afaR223 as well. 

They don't exactly match, what's the reason for the difference / should they be in one place?

Reply:
They can be same. Changed the default format to `DEFAULT_LOG_FORMAT` but removed all `[%n]`. `absl::StrReplaceAll(current_context->log_format_, {{"[%n]", ""}});`